frontend/lib/src/components/widgets/CameraInput/styled-components.ts

```diff
@@ -100,9 +100,9 @@ export const StyledImg = styled.img<StyledImgProps>(({ theme, opacity }) => ({
 }))
 
 export const StyledLink = styled.a(({ theme }) => ({
-  color: theme.colors.primary,
+  color: theme.colors.link,
+  textDecoration: "underline",
```

Comment:
We probably want to use the new config option here: `textDecoration: theme.linkUnderline ? "underline" : "none",` here

Reply:
Done.